File: day2.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Arithmetic {

    public static void main(String[] args) {
    
      double M = 10; // original meal price
      int T = 1; // tip percentage
      int X = 3; // tax percentage
      
      double valor = (double)M+M*X/100 + M*T/100;
       
      int total = (int) Math.round(valor);
      
      System.out.print("The final price of the meal is $"+total+".");
      
    }
}

File: JAVASCRIPT/day2.js
function processData(input) {
    var data = input.split('\n');
    var M = Number(data[0]);
    var tip = (M * Number(data[1])) / 100;
    var tax = (M * Number(data[2])) / 100;
    var total = (M + tip + tax).toFixed(0);
    console.log("The final price of the meal is $" + total + ".");
}